src/lib/datepicker/datepicker.ts

```diff
@@ -49,6 +49,11 @@ let datepickerUid = 0;
   selector: 'md-datepicker-content',
   templateUrl: 'datepicker-content.html',
   styleUrls: ['datepicker-content.css'],
+  host: {
+    '[class.mat-datepicker-content]': 'true',
+    '[class.mat-datepicker-content-touch]': 'datepicker.touchUi',
+    '[class.mat-datepicker-content-non-touch]': '!datepicker.touchUi',
```

Comment:
Why have two classes instead of a style without the `-touch` class and then another with (where the latter would inherently be higher specificity)?